### PR TITLE
Model size, CDNUrl and WebNN API spec link improvements

### LINKED ISSUE #533: [example] construct the modelCDNUrl instead of hand writing
For example https://github.com/intel/webml-polyfill/blob/master/examples/util/base.js#L30, are we able to construct the `modelCDNUrl` by `modelFile` and `windows.location`. That would be more flexible for deployment.

### LINKED ISSUE #534: [example] display the model size
Suggestion for discussion: display the model size while downloading and in model info page.

### LINKED ISSUE #535: [example] add link to webnn spec
Suggestion: add the link https://webmachinelearning.github.io/webnn/ to WebNN API spec. So users can refer to spec for use cases and API.

### PR DESCRIPTION
**Pull request by ibelem**
1. Fix #533 construct the modelCDNUrl by modelFile and windows.location
2. Fix #534 display the model size while downloading and in model info page
3. Fix #535 add the link https://webmachinelearning.github.io/webnn/ to WebNN API spec in pages footer
4. Update path 'posenet' to 'skeleton_detection'
5. Update path 'deeplab' to 'semantic_segmentation'
6. Other UI adjustments

### CODE REVIEW

**Comment by ibelem** [comment]
@pinzhenx @huningxin PTAL.

File: examples/util/base.js [hunk 1]
```diff
@@ -253,16 +256,39 @@ const ssdlite_mobilenetv2_tflite = {
...
+}
+  
+const deeplab_mobilenet_v2_513_dilated_tflite = {
+  modelName: 'deeplab_mobilenet_v2_513_dilated_tflite',
+  modelSize: '8.4MB',
+  modelFile: '../semantic_segmentation/model/deeplab_mobilenetv2_513_dilated.tflite',
+  labelsFile: '../semantic_segmentation/model/labels.txt',
+  inputSize: [513, 513, 3],
+  outputSize: [513, 513, 21],
+  intro: 'Extend DeepLabv3 to include a simple yet effective decoder module to refine the segmentation results especially along object boundaries. Furthermore, in this encoder-decoder structure one can arbitrarily control the resolution of extracted encoder features by atrous convolution to trade-off precision and runtime.',
```
Comment (pinzhenx) [inline examples/util/base.js]: I suggest changing the description to:
> Equivalent to the model above (without `dilated` suffix) but only available on platforms that natively support atrous convolution.

Comment (ibelem) [reply examples/util/base.js]: Thanks @pinzhenx , fixed in https://github.com/intel/webml-polyfill/pull/538/commits/2aa77f76375590abe7fdf53b30cd70f53b940d21

File: examples/util/base.js [hunk 2]
```diff
@@ -288,6 +314,12 @@ const humanPoseEstimationModels = [
   posenet,
 ];
 
+const semanticSegmentationModels = [
+  deeplab_mobilenet_v2_513_tflite,
```
[the remark this answers is not in the document]
Comment (ibelem) [reply examples/util/base.js]: Great suggestion, will do.

**Comment by pinzhenx** [comment]
LGTM, Thanks.
@huningxin Please help to merge this PR if there's no problem.

File: examples/util/base.js [hunk 3]
```diff
@@ -137,11 +138,11 @@ const mobilenet_v2_onnx = {
     softmax: true,
   },
   intro: 'MobileNetV2 improves the state of the art performance of mobile models.',
-  paperUrl: 'https://arxiv.org/abs/1801.04381',
-  modelCDNUrl: 'https://cdn_host_and_path/mobilenetv2-1.0.onnx'
+  paperUrl: 'https://arxiv.org/abs/1801.04381'
 };
 const resnet_v1_onnx = {
   modelName: 'resnet_v1_onnx',
+  modelSize: '46.8MB',
```
Comment (pinzhenx) [inline examples/util/base.js]: the size of resnet50 v1 is 102.6MB. This is the size of resnet18 v1.

File: examples/util/base.js [hunk 4]
```diff
@@ -156,11 +157,11 @@ const resnet_v1_onnx = {
     softmax: true,
   },
   intro: 'A residual learning framework to ease the training of networks that are substantially deeper than those used previously. This result won the 1st place on the ILSVRC 2015 classification task.',
-  paperUrl: 'https://arxiv.org/abs/1512.03385',
-  modelCDNUrl: 'https://cdn_host_and_path/resnet50v1.onnx'
+  paperUrl: 'https://arxiv.org/abs/1512.03385'
 };
 const resnet_v2_onnx = {
   modelName: 'resnet_v2_onnx',
+  modelSize: '46.8MB',
```
Comment (pinzhenx) [inline examples/util/base.js]: the size of resnet50 v2 is 102.4MB. This is the size of resnet18 v2.

File: examples/util/base.js (hunk 3 above)
Comment (pinzhenx) [reply examples/util/base.js]: I have once used the resnet18 but changed in https://github.com/intel/webml-polyfill/pull/353. You might need to update the local models

**Comment by ibelem** [comment]
Re-downloaded the resnet50 models from [Resnet-50 v1(Onnx)](https://s3.amazonaws.com/onnx-model-zoo/resnet/resnet50v1/resnet50v1.onnx)
10. [Resnet-18 v2(Onnx)](https://s3.amazonaws.com/onnx-model-zoo/resnet/resnet18v2/resnet18v2.onnx) and [Resnet-50 v2(Onnx)](https://s3.amazonaws.com/onnx-model-zoo/resnet/resnet50v2/resnet50v2.onnx). You are right, updated in https://github.com/intel/webml-polyfill/pull/538/commits/9f99821099a10325d2ecdd35e1cb6df4900307d0

**Comment by ibelem** [comment]
Thanks @pinzhenx , merge this PR.

**Pull request merged**